app/controllers/profile_controller.rb

```diff
@@ -11,4 +11,45 @@ def index
     @stories = Kaminari.paginate_array(get_stories(user))
                        .page(params[:page])
   end
+
+  def ban_user
+    ban_or_remove(true)
+  end
+
+  def remove_ban
```

Comment:
Would like to propose `unban_user` as the action name here. Or alternatively, renaming `ban_user` to `add_ban`.